src/common/Pagination/index.js

```diff
@@ -0,0 +1,45 @@
+import {
+  ArrowLeft,
+  ArrowRight,
+  Button,
+  ButtonText,
+  PageCounter,
+  PageNumber,
+  Wrapper,
+} from "./styled";
+
+const Pagination = ({
+  currentPage,
+  totalPages,
+  onFirstPage,
+  onPrevPage,
+  onNextPage,
+  onLastPage,
+}) => {
+  return (
+    <Wrapper>
+      <Button onClick={onFirstPage} disabled={currentPage === 1}>
+        <ArrowLeft />
+        <ButtonText>First</ButtonText>
+      </Button>
+      <Button onClick={onPrevPage} disabled={currentPage === 1}>
+        <ArrowLeft />
+        <ButtonText>Previous</ButtonText>
+      </Button>
+      <PageCounter>
+        Page <PageNumber>{currentPage}</PageNumber>
+        of <PageNumber>{totalPages}</PageNumber>
+      </PageCounter>
+      <Button onClick={onNextPage} disabled={currentPage === totalPages}>
+        <ArrowRight />
+        <ButtonText>Next</ButtonText>
+      </Button>
+      <Button onClick={onLastPage} disabled={currentPage === totalPages}>
+        <ArrowRight />
+        <ButtonText>Last</ButtonText>
```

Comment:
You should switch Arrow and Text places :)

Reply:
Is ready :-)